src/main/java/com/nike/cerberus/command/CerberusCommand.java

```diff
@@ -114,7 +132,8 @@ public String getEnvironment() {
                         EnvironmentalVarEnvironment;
 
         if (StringUtils.isBlank(calculatedEnv)) {
-            throw new IllegalArgumentException("Failed to determine environment, checked 'CERBERUS_CLI_ENV' env var and -e, --env, --environment command options, options must go before the command");
+            log.error("Failed to determine environment, checked 'CERBERUS_CLI_ENV' env var and -e, --env, --environment command options, Will attempt to use the AWS Default region (us-west-2");
+            calculatedEnv = Regions.DEFAULT_REGION.toString();
```

Comment:
Confused by environment vs. region in the same sentence?  Seems like copy/paste error?